one_leg:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 30

  leg_l_controller:
    type: position_controllers/JointPositionController
    joint: l_joint
    pid: {p: 1.0, i: 1.0, d: 0.0}
  leg_h_controller:
    type: position_controllers/JointPositionController
    joint: h_joint
    pid: {p: 1.0, i: 1.0, d: 0.0}
